odoo/fields.py

```diff
@@ -2561,6 +2561,9 @@ def _remove_inverses(self, records, value):
 
     def _update_inverses(self, records, value):
         """ Add `records` to the cached values of the inverse fields of `self`. """
+
+        if not value:
+            return
```

Comment:
Please leave out the blank line.
```suggestion
        if not value:
            return
```